docs/modules/ROOT/pages/kafka/kafka-reactive-binder/reactive_observability.adoc

```diff
@@ -0,0 +1,70 @@
+[[reactive-kafka-binder-observability]]
+= Observability in Reactive Kafka Binder
+
+In this section, we will describe how Micrometer based observability is enabled in the reactive Kafka binder.
+
+There is built in support for observability when it comes to producer binding, but you need to opt-in for this by enabling the following property.
```

Comment:
I thought `built-in` comes as a one word.
But I won't mind since I'm still learning English 😄